commands.py

```diff
@@ -273,10 +272,8 @@ def handle_privmsg(state: mantatail.ServerState, user: mantatail.UserConnection,
     if user not in channel.users:
         error_not_on_channel(user, receiver)
     else:
-        for usr in channel.users:
-            if usr.nick != user.nick:
-                message = f"PRIVMSG {receiver} :{privmsg}"
-                usr.send_que.put((message, user.get_user_mask()))
+        privmsg_message = f"PRIVMSG {receiver} :{privmsg}"
+        channel.queue_message_to_chan_users(privmsg_message, user, False)
```

Comment:
If you use a keyword argument, the reader doesn't have to guess what `False` does:

```suggestion
        channel.queue_message_to_chan_users(privmsg_message, user, send_to_self=False)
```

Reply:
Ah, nice! I didn't know that you could use the keyword in the argument too. I thought it was only in the parameters.